docs/reference/mapping.asciidoc

```diff
@@ -68,6 +68,14 @@ via the <<multi-fields>> parameter.
 [float]
 === Settings to prevent mappings explosion
 
+Defining too many fields in an index or having too many nested documents or too many fields that contain nested
```

Comment:
you are right, jut to be 100% sure, if I nested document introduce new fields, this can be a similar case of Defining too many fields?